conans/client/profile_loader.py

```diff
@@ -225,13 +225,6 @@ def get_package_name_value(item):
     if doc.options:
         base_profile.options.update(OptionsValues.loads(doc.options))
 
-    # The env vars from the current profile (read in doc)
-    # are updated with the included profiles (base_profile)
-    # the current env values has priority
-    current_env_values = EnvValues.loads(doc.env)
-    current_env_values.update(base_profile.env_values)
-    base_profile.env_values = current_env_values
-
```

Comment:
https://github.com/conan-io/conan/pull/9348/files#diff-711453b1a75934b6fa7f423dc82a3363a385a278e45ab7516105c67c60a82edfR162-R165

Should we remove the section [env] from there?

Reply:
The link doesn't point to a line in the diff, could you clarify it?